assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -799,7 +799,8 @@ def can_apply_ort(args: Namespace, logger):
         logger.warning("Enabling ORT has a breaking change with summarization and translation tasks "
                        "so diabling ORT for SUMMARIZATION and TRANSLATION tasks")
         return False
-    return args.apply_ort
```

Comment:
This will need updating the UTs as well. We can do it as a fast follow-up

Reply:
Yes will do